### ISSUE TITLE
Tasklist won't populate if main tasklist is empty

### ISSUE THREAD

**Issue by wallace11** [itchyny/calendar.vim#175]
Steps to reproduce:

0. Clear cache
1. Delete all lists, leaving only main list
2. Delete all tasks from main list until it shows "A fresh start" message (web version)
3. Make sure there are no completed tasks either
4. Tasks list will not populate in vim
5. Add a new task list
6. Still nothing
7. Add a new item to the new tasklist
8. Nothing
9. Add items to main task list
10. Now everything works
11. Delete items from main task list
12. Everything still shows up but can't add nor edit anything

**tldr;**
Nothing will show on tasklist unless there are items in it.

**Comment by itchyny** [comment]
Fixed recently.
